astropy/stats/tests/test_funcs.py

```diff
@@ -9,12 +9,16 @@
 from numpy.testing import assert_equal
 from numpy.testing.utils import assert_allclose
 
+from astropy.utils import minversion
+
 try:
     import scipy  # pylint: disable=W0611
 except ImportError:
     HAS_SCIPY = False
+    SCIPY_LT_1_4 = True
 else:
     HAS_SCIPY = True
+    SCIPY_LT_1_4 = not minversion(scipy, '0.14.0')
```

Comment:
Sorry to bother you again with this line. Was it intentional you named it `*_1_4` instead of `*_0_14`? I forgot to explain it in https://github.com/astropy/astropy/pull/5271#discussion_r76594551 but the current name suggests SciPy 1.4 instead of the tested 0.14.